### PR TITLE
Saving progress

### PR DESCRIPTION
**Pull request by ycoya**
A way to save password generated and be able to continue from where we left of. So we can stop and continue without restarting all progress. Tested in terminal command laravel framework.

        $charRecorded = "w1b";
        $this->count = 15656;
        BruteForceAttacker::bootUp($charRecorded);

        for ($i = $charRecorded ? strlen($charRecorded): 1 ; $i <= 5 ;$i++) {

            BruteForceAttacker::run([
                'length' => $i,
                'charMap' => array_merge(range('0', '9'), range('a', 'z')),
                'callback' => [$this, 'sendTelnetCmd'],
                'startsCount' => $this->count,
            ]);
            if($this->found) {
                break;
            }
        }

    }

    public function sendTelnetCmd($string, $count)
    {
        $this->info("string: $string, count: $count");
        $this->count = $count;
        if ($string=="a2b") {
            $this->info( "Matched `{$string}` with {$count} times");
            return $this->found = true;
        }

    }
`

### CODE REVIEW

File: src/BruteForceAttacker.php [hunk 1]
```diff
@@ -70,13 +76,23 @@ public static function run($options)
         self::$callback = $options['callback'];
         self::$skipLength = $options['skipLength'];
         self::$skipCount = $options['skipCount'];
-        self::$count = 0;
+        self::$count = $options['startsCount'];
 
         // Run
-        $length = ($options['length'] >= 1) ? intval(floor($options['length'])) : 1;
+        $length = ($options['length'] >= 1) ?  intval(floor($options['length'])) : 1;
```
Comment (yidas) [inline src/BruteForceAttacker.php:82]: Could you revert this change? 
Thanks

Comment (ycoya) [reply src/BruteForceAttacker.php:82]: Sure!, you mean the white space before `intval(...)`.

Comment (ycoya) [reply src/BruteForceAttacker.php:82]: Well, it seems you have already done it.
